woocommerce/woocommerce
# Update phpcs to squizlabs/php_codesniffer

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR updates the PHP code sniffer to [squizlabs/php_codesniffer](https://github.com/PHPCSStandards/PHP_CodeSniffer/tree/master).

Closes #41475 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run `composer i` in the `plugins/woocommerce` folder
2. Run `vendor/bin/phpcs src`
3. This should log a significant list of `phpcs` warnings and errors
4. Run `grep -nR license ./` in the `plugins/woocommerce/bin/composer/phpcs` folder
5. The licenses should reference `https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/licence.txt`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
